migrations/Version202209131056091465_qtiItemPci.php

```diff
@@ -33,7 +33,7 @@ public function up(Schema $schema): void
     public function down(Schema $schema): void
     {
         throw new IrreversibleMigration(
-            'In order to undo this migration, please revert the client-side changes and run ' . RegisterPciMathEntry::class
+            'To undo this migration, please revert the client-side changes and run' . RegisterPciMathEntry::class
```

Comment:
**chore:** I think the last space should remain, otherwise the class name will be glued with the word `run` :)